test/metabase/lib/drill_thru_test.cljc

```diff
@@ -439,6 +441,43 @@
                     (lib/available-drill-thrus query -1 context)))
             (test-drill-applications query context)))))))
 
+(deftest ^:parallel table-view-available-drill-thrus-aggregate-column-header-test
+  (let [query (-> (lib/query meta/metadata-provider (meta/table-metadata :orders))
+                  (lib/aggregate (lib/count))
+                  (lib/aggregate (lib/sum (meta/field-metadata :orders :tax)))
+                  (lib/aggregate (lib/max (meta/field-metadata :orders :discount)))
+                  (lib/breakout (meta/field-metadata :orders :product-id))
+                  (lib/breakout (-> (meta/field-metadata :orders :created-at)
+                                    (lib/with-temporal-bucket :month))))]
```

Comment:
This query is copied directly from

https://github.com/metabase/metabase/blob/5ab317540d064ac3d402508d0038629e8a041575/frontend/src/metabase-lib/drills.unit.spec.ts#L115-L161

I don't really love that we have unit tests that hardcode legacy MBQL queries in the frontend, it would make more sense if the FE unit tests created queries with MLv2 as well